ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOModelTestSuite.scala

```diff
@@ -527,6 +527,70 @@ class H2OMOJOModelTestSuite extends FunSuite with SharedH2OTestContext with Matc
     mojo.getDefaultThreshold() shouldBe 0.5
   }
 
+  test("getCrossValidationScoringHistory returns histories when they are available") {
```

Comment:
I think "cross-validation scoring history" is a misleading name, I wouldn't know how to intuitively interpret this.

The output of this method seems to be the scoring history of "cross-validation models" (each of k-fold models).

The main issue here is that it could be interpreted of the scoring history of cross-validation itself (eg. training all cv models in concert and calculating cv metrics after each scoring iteration) - we don't have this.

cc: @tomasfryda

Reply:
@michalkurka This was introduced with the learning curve PR so we can make learning curves even for the CV models (thus `cross_validation_scoring_history` stores the scoring history of individual CV models) which can be helpful when a user doesn't provide `validation_frame`.

I don't mind changing it. Do you think `cross_validation_models_scoring_history` or `cv_models_scoring_history` would be clearer?

Reply:
@tomasfryda I think this is fine on h2o side because it is internal afaik to (auto-)explain

however, in the context of SW I find this name confusing - there are other ways to make it clearer - eg. make it a map of model key to scoring history

Reply:
As concluded in (other) discussion, CrossValidationModelsScoringHistory will be utilized.